keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -0,0 +1,442 @@
+    )(x)
+    x = layers.BatchNormalization(
+        axis=channel_axis,
+        epsilon=1e-3,
+        momentum=0.999,
+        name=prefix + "project/BatchNorm",
+    )(x)
+
+    if stride == 1 and infilters == filters:
+        x = layers.Add(name=prefix + "Add")([shortcut, x])
+
+    return x
+
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class MobileNetV3Backbone(Backbone):
+    """Instantiates the MobileNetV3 architecture.
+
+    References:
+        - [Searching for MobileNetV3](https://arxiv.org/pdf/1905.02244.pdf)
+        (ICCV 2019)
+        - [Based on the Original keras.applications MobileNetv3]
+        (https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v3.py)
+
+    For transfer learning use cases, make sure to read the [guide to transfer
+        learning & fine-tuning](https://keras.io/guides/transfer_learning/).
+
+    Args:
+        stack_fn: a function that returns tensors passed through Inverted
+            Residual Blocks.
+        last_point_ch: integer, the number of filters for the convolution layer.
+        include_rescaling: bool, whether to rescale the inputs. If set to True,
+            inputs will be passed through a `Rescaling(scale=1 / 255)`
+            layer.
+        input_shape: optional shape tuple, defaults to (None, None, 3).
+        input_tensor: optional Keras tensor (i.e., output of `layers.Input()`)
+            to use as image input for the model.
+        alpha: float, controls the width of the network. This is known as the
+            depth multiplier in the MobileNetV3 paper, but the name is kept for
+            consistency with MobileNetV1 in Keras.
+            - If `alpha` < 1.0, proportionally decreases the number
+                of filters in each layer.
+            - If `alpha` > 1.0, proportionally increases the number
+                of filters in each layer.
+            - If `alpha` = 1, default number of filters from the paper
+                are used at each layer.
+        minimalistic: in addition to large and small models, this module also
+            contains so-called minimalistic models; these models have the same
+            per-layer dimensions characteristic as MobilenetV3 however, they
+            don't utilize any of the advanced blocks (squeeze-and-excite units,
+            hard-swish, and 5x5 convolutions). While these models are less
+            efficient on CPU, they are much more performant on GPU/DSP.
+        dropout_rate: a float between 0 and 1 denoting the fraction of input
+            units to drop, defaults to 0.2.
+
+    Examples:
+    ```python
+    input_data = tf.ones(shape=(8, 224, 224, 3))
+
+    # Randomly initialized backbone with a custom config
+    model = MobileNetV3Backbone(
+        stack_fn=None,
+        last_point_ch=1024,
+        include_rescaling=False,
+    )
+    output = model(input_data)
+    ```
+    """
+
+    def __init__(
+        self,
+        *,
+        stack_fn,
```

Comment:
Let's use a string param rather than a callback like we did for resnet ([link](https://github.com/keras-team/keras-cv/blob/055b01be6fb9da16a94ae06cb14601b40683b005/keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py#L230)).

Reply:
I would also make this a named arg rather than a positional one.

Reply:
Addressed